survlime/survlime_explainer.py

```diff
@@ -264,4 +274,68 @@ def solve_opt_problem(
         prob = cp.Problem(objective)
         result = prob.solve(verbose=verbose)
         cox_coefficients = b.value[:, 0]
+
+        self.computed_weights = cox_coefficients
         return cox_coefficients
+
+    def plot_weights(self, figsize: Tuple[int, int] = (10, 10),
+                     feature_names: List[str] = None,
+                     scale_with_data_point: bool = False) -> None:
+        """Plot the weights of the computed model using 
+            seaborn as plotting library
+        Args:
+            figsize (Tuple[int, int]): size of the figure
+        """
+        if self.computed_weights is None:
+            raise ValueError("SurvLIME weights not computed yet")
+        
+        # Check
+        elif feature_names is not None:
+            feature_names = feature_names
+        else:
+            feature_names = self.feature_names
+
+        if scale_with_data_point:
+            weights = self.computed_weights * self.data_point
+        else:
+            weights = self.computed_weights
+
+        _, ax = plt.subplots(figsize=figsize)
+
+        # sort weights in descending order
+        sorted_weights = np.sort(weights)[::-1]
+        # sort feature names so that they match the sorted weights
+        sorted_feature_names = [feature_names[i] for i in np.argsort(weights)[::-1]]
```

Comment:
We are sorting twice: one for `sorted_weights` and another for `sorted_feature_names`. I think we can sort just once